roles/ansible_service_broker/tasks/install.yml

```diff
@@ -72,6 +72,12 @@
       - apiGroups: ["image.openshift.io", ""]
         resources: ["images"]
         verbs: ["get", "list"]
+      - apiGroups: ["network.openshift.io"]
+        resources: ["clusternetworks", "netnamespaces"]
+        verbs: ["get"]
+      - apiGroups: ["network.openshift.io"]
+        resources: ["netnamespaces"]
+        verbs: ["update"]
```

Comment:
What updates does the ASB make, and are they user-driven?

Reply:
The ASB needs to join the networks of the transient namespace that he APB runs into the network of the target namespace that we are deploying to. The ASB will also remove the transient namespace from this network, if the namespace is kept, for whatever reason. This is so APB's can interact with the network of the target namespace.

Reply:
@enj can you explain to me what the issue is with the bidirectional access and the secrets. 
Just having network access to reach routes/services still does not give the user access to reach the resources in the transient namespace right? Sorry if I am misunderstanding the interaction model here.

Reply:
@shawn-hurley my concern is that I do not know what level of visibility this gives the user into the transient namespace.  Perhaps the bundle or the ASB uses the AWS secrets over the network in a way that a malicious user could see them?  I am not saying a vulnerability exists; I am just saying that I cannot say this operation is guaranteed to be safe.

Reply:
@enj @liggitt 

Did our discussion in this PR cover your concerns? 
https://github.com/openshift/ansible-service-broker/pull/600

Reply:
yes